test/agent/go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/aws/amazon-vpc-cni-k8s/test/agent
 
-go 1.14
+go 1.17
 
 require (
 	github.com/coreos/go-iptables v0.6.0
 	github.com/vishvananda/netlink v1.1.0
 	golang.org/x/sys v0.0.0-20210426230700-d19ff857e887
 )
+
+require github.com/vishvananda/netns v0.0.0-20191106174202-0a2b9b5464df // indirect
```

Comment:
Latest not compatible with go 1.17?

Reply:
This line got inserted upon `go mod tidy`, it seems like both https://github.com/vishvananda/netlink and github.com/vishvananda/netns specify go 1.12 version.